include/glow/Partitioner/Partitioner.h

```diff
@@ -317,10 +315,10 @@ class Partitioner {
   void dumpDAG(llvm::StringRef dotFilename) const;
 
   /// Get function for computeTime_
-  ComputeTimeMapTy getComputeTime() const { return computeTime_; }
+  float getComputeTime(Node *N) const { return computeTime_.at(N); }
 
   /// Get function for memUsage_
-  MemUsageMapTy getMemUsage() const { return memUsage_; }
+  uint64_t getMemUsage(Node *N) const { return memUsage_.at(N); }
```

Comment:
nit: const Node here and line 318.

Reply:
It looks like I can't use a `const Node *` to look something up in a `map<Node *, ...>`.  Not without a `const_cast`, anyways.